cabal-install/Distribution/Client/CmdLegacy.hs

```diff
@@ -112,9 +112,9 @@ deprecationNote :: String -> String
 deprecationNote cmd =
     "The " ++ cmd ++ " command is a part of the legacy v1 style of cabal usage.\n\n" ++
 
-    "Please switch to using either the new project style and the ++ new-" ++ cmd ++ 
-    "command or the legacy v1-" ++ cmd ++ "\nalias as new-style projects will\n" ++
-    "become the default in the next version of cabal-install. Please file a\n" ++
+    "Please switch to using either the new project style and the new-" ++ cmd ++ 
+    " command or the legacy v1-" ++ cmd ++ "alias as new-style projects will" ++
```

Comment:
That needs a space, or else we get `v1-buildalias`.

Reply:
Good spot, thanks.